docs/networkfence.md

```diff
@@ -0,0 +1,47 @@
+# NetworkFence
+
+NetworkFence is a cluster-scoped custom resource that allows Kubernetes to invoke "Network fence" operation on a storage provider.
+
+The user needs to specify the kind of operation they want to perform (fence/unfence), and then provide the list of CIDR blocks on which the mentioned operation is to be performed.
+This results in the fence/unfence of the target clients via NetworkFence custom resource.
+
+### Fence Operation
+
+```yaml
+apiVersion: csiaddons.openshift.io/v1alpha1
+kind: NetworkFence
+metadata:
+  name: network-fence-sample
+spec:
+  operation: fence
```

Comment:
how you will send a request to a specific CSI driver what is the identifier here? suppose 2 drivers are registered for fencing how a user can specify to which CSI driver he needs to invoke for fencing?

Reply:
Thanks for pointing that out. Updated now.